utils/sparse.py

```diff
@@ -40,14 +41,19 @@ def check_download_sparsezoo_weights(path):
 
 
 class SparseMLWrapper(object):
-    def __init__(self, model, checkpoint_recipe, train_recipe, steps_per_epoch=-1):
+    def __init__(self, model, checkpoint_recipe, train_recipe, steps_per_epoch=-1, one_shot=False):
         self.enabled = bool(train_recipe)
         self.model = model.module if is_parallel(model) else model
         self.checkpoint_manager = ScheduledModifierManager.from_yaml(checkpoint_recipe) if checkpoint_recipe else None
         self.manager = ScheduledModifierManager.from_yaml(train_recipe) if train_recipe else None
         self.logger = None
         self.start_epoch = None
         self.steps_per_epoch = steps_per_epoch
+        self.one_shot = one_shot
```

Comment:
I have been thinking @rahul-tuli, would it be a good idea to add (somewhere) a quick definition of what actually "one-shot" is? This would allow avoiding repeated discussions (like the one we had last time) in the future. "One-shot" in our context is quite different from how it is broadly used in ML community, I can imagine this can be the source of misunderstandings. Definitely, we should define it in the upcoming docs.

Reply:
That's a very good thought! I'll keep a note of this and add it in docs